### ISSUE TITLE
Automatically set labelId property to existing <label> element

### ISSUE THREAD

**Issue by RichardDavies** [Choices-js/Choices#1300]
**Is your feature request related to a problem? Please describe.**
The `labelId` property exists that allows you to add `aria-labelledby` to the choices element for improved accessibility. This is great, but this property must be manually set for each Choices element, otherwise the element doesn't have an associated label as [required for accessibility](https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html).

**Describe the solution you'd like**
Instead of defaulting to no label, it would be ideal if this property would default to setting `aria-labelledby` to the `<label>` element associated with the `<select>` element.

For example, assume you're creating a Choices element for the following HTML:

```html
<label for="choices-multiple-remove-button">With remove button</label>
<select
   class="form-control"
   name="choices-multiple-remove-button"
   id="choices-multiple-remove-button"
   placeholder="This is a placeholder"
   multiple
>
   <option value="Choice 1" selected>Choice 1</option>
   <option value="Choice 2">Choice 2</option>
   <option value="Choice 3">Choice 3</option>
   <option value="Choice 4">Choice 4</option>
</select>
```

Since the existing `<label>` element doesn't have an `id`, you could assign it one like "choices-multiple-remove-button_choiceslabel" and then set the `labelId` property to "choices-multiple-remove-button_choiceslabel" as the default value.

**Additional context**
Choices should maintain any of the accessibility information, such as form field labels, that exist in the native HTML elements when creating the enhanced Choices element.
